### PR TITLE
add auxtel-{archive,ccs} clusters

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: cluster/auxtel-archive.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+# XXX these values should be migrated to the site layer
+tuned::profile: "latency-performance"
+selinux::mode: "disabled"
+firewall::ensure: "stopped"
```
Comment (adrienthebo) [inline cluster/auxtel-archive.yaml]: :-1: for site-wide firewall disable; we need to distribute reasonable rules that permit `139.229/16` and `198./8`, in the future but I'm quite leery of using this as a site wide default.

File: cluster/auxtel-archive.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+puppet_agent::package_version: "6.7.0"
+puppet_agent::collection: "puppet6"
```
Comment (adrienthebo) [inline cluster/auxtel-archive.yaml]: :+1: for site-wide puppet version management.

File: cluster/auxtel-archive.yaml (hunk 1 above)
Comment (jhoblitt) [reply cluster/auxtel-archive.yaml]: I think it should be off as the default unless explicitly set at a more specific layer as a safe guard against getting locked out of not-completely-configured systems.

File: cluster/auxtel-archive.yaml (hunk 2 above)
Comment (jhoblitt) [reply cluster/auxtel-archive.yaml]: How about we sit down next week and cowork through these sort of cleanups?

**Pull request merged**